### DISCUSSION TITLE
How To Auto-Update Tap Formula On New Release?

### DISCUSSION THREAD

**Discussion by Blackjacx** [Homebrew/discussions#2558] [Tap maintenance and Homebrew development]
Hey there 👋

I wanted to ask you, if there is an existing, automatic way to update my [homebrew formula/tap](https://github.com/ChargePoint/homebrew-xcparse/blob/master/Formula/xcparse.rb) whenever I create a new release for [my CLI tool](https://github.com/Blackjacx/Assist)?

Especially interesting would be if there is a tool that can automatically update the formula ruby file with the new tag, similar to ` brew bump-formula-pr asc --tag 0.1.1` which does this for homebrew-core though. 

Best 🙂

**Comment by carlocab** [comment]
Try https://github.com/dawidd6/action-homebrew-bump-formula.

**Comment by jtbandes** [comment]
There's also [mislav/bump-homebrew-formula-action](https://github.com/mislav/bump-homebrew-formula-action).

<img src="https://api.star-history.com/svg?repos=mislav/bump-homebrew-formula-action,dawidd6/action-homebrew-bump-formula&type=Date" width="600">

**Reply by carlocab** [reply]
I'm a big fan of `@mislav`'s work, but in my experience, `@dawidd6`'s action does things better. I haven't taken a look at the code in a while, but it's likely due to `@dawidd6`'s action relying more heavily on `brew` to do things (e.g. through `livecheck` or `bump-formula-pr`) rather than on inventing a different solution.
